sdk/core/azure-core/test/ut/environment_log_level_listener_test.cpp

```diff
@@ -144,8 +133,9 @@ TEST_F(EnvironmentLogLevelListenerTest, GetLogListenerVerbose)
   std::string text = buffer.str(); // text will now contain "Bla\n"
   auto listener = EnvironmentLogLevelListener::GetLogListener();
 
-  listener(Logger::Level::Verbose, "message");
   EXPECT_NE(listener, nullptr);
+
+  listener(Logger::Level::Verbose, "message");
```

Comment:
why switch the location of the listener invocation ?

Reply:
Clearer test logs. We now first verify it is not null (`EXPECT_NE(listener, nullptr)`), and only after that we call it. Before, we call first, check for null after.